web/packages/teleport/src/components/EventRangePicker/Custom/Custom.jsx

```diff
@@ -1,168 +0,0 @@
-/*
- * Teleport
- * Copyright (C) 2023  Gravitational, Inc.
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-
-import React from 'react';
-import { isAfter, endOfDay, startOfDay, isSameDay, subMonths } from 'date-fns';
-import styled from 'styled-components';
-import dayPicker from 'react-day-picker/DayPicker';
-import 'react-day-picker/lib/style.css';
-import { Flex } from 'design';
-import { Cross as CloseIcon } from 'design/Icon';
-
-// There is a vite issue with react-day-picker in production builds
-// https://github.com/vitejs/vite/issues/2139
-// TODO(ryan): After node v18 upgrade, swap to the Vite config approach instead of this one
-// also, we should look into upgrading react-day-picker
```

Comment:
i tested a build (`make full`), and i had no issues, it looks like it was resolved